.ci.yaml

```diff
@@ -25,8 +25,29 @@ platform_properties:
         ]
       device_type: none
       os: Windows
+  mac_x64:
+    properties:
+      dependencies: >-
+        [
+          {"dependency": "xcode", "version": "14a5294e"},
+          {"dependency": "gems", "version": "v3.3.14"}
+        ]
+      os: Mac-12
+      device_type: none
+      cpu: x86
+      xcode: 14a5294e # xcode 14.0 beta 5
+      
 
 targets:
+  - name: Mac_x64 darwin_lint_podspecs
```

Comment:
We should preserve the TODO from the Cirrus config about moving this to ARM.

Reply:
Good catch! Added it back.